crates/proof-of-sql/src/sql/parse/query_expr_tests.rs

```diff
@@ -1673,7 +1711,7 @@ fn varchar_column_is_not_allowed_within_numeric_aggregations() {
             if expression == "cannot use expression of type 'varchar' with numeric aggregation function 'sum'"
     ));
 
-    let max_query = "select max(s) from sxt.t";
+    let max_query = "select max(name) from sxt.employees";
```

Comment:
What about `max(salary)`?

Reply:
No, the change I made is in relevance to test `varchar_column_is_not_allowed_within_numeric_aggregations` because the test is checking for the error `cannot use expression of type 'varchar' with numeric aggregation function 'max'` to be logged

Reply:
Same here, this seems fine to me.